projects/demo-playwright/utils/page-objects/input-card-grouped.po.ts

```diff
@@ -0,0 +1,42 @@
+import {Locator, Page} from '@playwright/test';
+
+export class TuiInputCardGroupedPO {
+    readonly cardGroupInput = this.page.locator(
+        `[automation-id="tui-input-card-grouped__card"]`,
+    );
+
+    readonly cleanerIcon = this.page.locator(
+        `[automation-id="tui-input-card-grouped__cleaner"]`,
+    );
+
+    readonly disableFormControlToggle = this.page
+        .locator(`tr`)
+        .filter({
+            has: this.page.locator(`[automation-id="tui-documentation__property-name"]`, {
+                hasText: `disabled`,
+            }),
+        })
+        .locator(`[automation-id="tui-toggle__checkbox"]`);
```

Comment:
This method to disable component is just legacy way (when API pages did not support query param for `disable`).
Now we can just add it to url:
https://taiga-ui.dev/components/input-card-grouped/API?disabled=true

```ts
await tuiGoto(page, `components/input-card-grouped/API?disabled=true`);
```

____

Delete this property from `TuiInputCardGroupedPO` and update tests, please